## Chapter 4: Creating a Binomial Probability Distribution

<b> Q1. What are the parameters of the binomial distribution for the probability of rolling either a 1 or a 20 on a 20-sided die, if we roll the die 12 times?

In [1]:
#k outcomes
#n trials
#p probability
n = 12
k = 1
p = 2/20

<b>Q2. There are four aces in a deck of 52 cards. If you pull a card, return the card, then reshuffle and pull a card again, how many ways can you pull just one ace in five pulls?

In [2]:
choose(5,1)

[1] 5

<b> Q3. For the example in question 2, what is the probability of pulling five aces in 10 pulls (remember the card is shuffled back in the deck when it is pulled)?

In [5]:
#Disable Scientific Notation
options(scipen = 999)
(4/52)^4

[1] 0.00003501278

In [7]:
1/32000

[1] 0.00003125

In [10]:
pbinom(5,10,1/13,lower.tail = FALSE)

[1] 0.00003313921

<b>Q4. When you’re searching for a new job, it’s always helpful to have more than one offer on the table so you can use it in negotiations. If you have a 1/5 probability of receiving a job offer when you interview, and you interview with seven companies in a month, what is the probability you’ll have at least two competing offers by the end of that month?

In [11]:
pbinom(1,7,1/5,lower.tail = FALSE)

[1] 0.4232832

42% chance of two or more job offers if you interview at 7 companies

<b>Q5. You get a bunch of recruiter emails and find out you have 25 interviews lined up in the next month. Unfortunately, you know this will leave you exhausted, and the probability of getting an offer will drop to 1/10 if you’re tired. You really don’t want to go on this many interviews unless you are at least twice as likely to get at least two competing offers. Are you more likely to get at least two offers if you go for 25 interviews, or stick to just 7?

In [14]:
pbinom(1,7,1/5,lower.tail = FALSE)
pbinom(1,25,1/10,lower.tail =FALSE)

[1] 0.4232832

[1] 0.7287941

The probability of getting 2 offers in 25 interviews is nearly 73%

In [16]:
0.728794093538633/0.4232832

[1] 1.721765

1.72 times more likely to get two or more offers